### PR TITLE
Query f_{modified,ruleset,version} on match {start,end}

### PR DESCRIPTION
**Pull request by osm**

### CODE REVIEW

File: src/match.c [hunk 1]
```diff
@@ -2784,6 +2801,21 @@ void PlayerReady(qbool startIdlebot)
...
+		{
+			stuffcmd(self, "say f_modified\n");
+		}
+
+		if (cvar("k_on_start_f_ruleset"))
+		{
+			stuffcmd(self, "say f_ruleset\n");
+		}
+
+		if (cvar("k_on_start_f_version"))
```
Comment (dsvensson) [inline src/match.c:2814]: third time's the charm?

Comment (dsvensson) [reply src/match.c:2814]: oh, I'm blind :D

**Review by dsvensson — APPROVED** [review]

File: src/match.c (hunk 1 above)
Comment (osm) [reply src/match.c:2814]: The `f_ruleset` check truncates the version string, so `f_version` is unfortunately needed as well :(

**Comment by tcsabina** [comment]
Hi @osm,

Please give us some details about this PR if you want it to be merged. Let me not guess from the source ;)

**Comment by osm** [comment]
Hi @tcsabina,

I thought this wasn't required since it was a relatively small and obvious change. However, as I mentioned in #dev-corner, this seemed like the least intrusive way to embed client and ruleset information in MVDs without forcing everyone to update their clients (which would likely be very difficult to achieve).

**Pull request merged**